server/src/internalClusterTest/java/org/elasticsearch/action/bulk/IncrementalBulkIT.java

```diff
@@ -65,7 +65,7 @@ protected Settings nodeSettings(int nodeOrdinal, Settings otherSettings) {
             .put(super.nodeSettings(nodeOrdinal, otherSettings))
             .put(IndexingPressure.SPLIT_BULK_LOW_WATERMARK.getKey(), "512B")
             .put(IndexingPressure.SPLIT_BULK_LOW_WATERMARK_SIZE.getKey(), "2048B")
-            .put(IndexingPressure.SPLIT_BULK_HIGH_WATERMARK.getKey(), "2KB")
+            .put(IndexingPressure.SPLIT_BULK_HIGH_WATERMARK.getKey(), "4KB")
             .put(IndexingPressure.SPLIT_BULK_HIGH_WATERMARK_SIZE.getKey(), "1024B")
             .build();
     }
```

Comment:
There are tests in this class `testIncrementalBulkLowWatermarkBackOff` and `testIncrementalBulkHighWatermarkBackOff`.  Can we modify them to ensure that `IndexingPressure indexingPressure` properly accounts for the splits? We check indexing pressure stats for other things in this IT so it should be a small addition.